tests/test_cancellation_player.py

```diff
@@ -0,0 +1,88 @@
+from unittest import TestCase
+from table import Table
+from bet import Bet
+from outcome import Outcome
+from players.cancellation_player import PlayerCancellation
+
+
+class TestPlayerCancellation(TestCase):
+    def setUp(self):
+        self.table = Table()
+        self.player_cancellation = PlayerCancellation(self.table)
+
+    def test_resetSequence_resets_sequence(self):
+        self.player_cancellation.sequence = []
+        expected_seq_after_resetSequence = [1, 2, 3, 4, 5, 6]
+        self.player_cancellation.resetSequence()
+        self.assertEqual(
+            self.player_cancellation.sequence, expected_seq_after_resetSequence
+        )
+
+    def test_lose_adds_bet_amount_to_sequence(self):
```

Comment:
better rename to `bet_amount_added_to_sequence_after_lose`. This way test name represents a behavior. The current test name shows which method (lose) updates the sequence, this is implementation details.

Reply:
Done.